apps/cowswap-frontend/src/legacy/state/orders/consts.ts

```diff
@@ -18,3 +18,7 @@ export const OUT_OF_MARKET_PRICE_DELTA_PERCENTAGE = new Percent(1, 100) // 1/100
 
 // Clear order's storage
 export const MAX_ITEMS_PER_STATUS = 10
+// In order to keep expired orders displayed at least for 7 days more after expiration
+export const ORDER_VALIDITY_THRESHOLD = ms`7d`
+// In order to keep expired but filled orders displayed at least for 1 month more after expiration
+export const FILLED_ORDER_VALIDITY_THRESHOLD = ms`30d`
```

Comment:
I think this is too aggressive.
Even if the order expired, I would like to see them it in my history for example to check which prices I used last.

Same thing for filled orders.
I suggest 1y for both.